cmd/crd/main.go

```diff
@@ -14,7 +14,8 @@ import (
 
 func main() {
 	if err := innerMain(); err != nil {
-		log.Fatal(err)
+		fmt.Fprintf(os.Stderr, "%#v", err)
```

Comment:
why don't we use zap everywhere? isn't it initialized yet by this point?

Reply:
No, this is main. What are you going to do if logger initialization fails and you need to print the error?